### PR TITLE
sql: update docs for `explain analyze`

### PR DESCRIPTION
**Pull request by qw4990**
### What is changed, added or deleted? 
explain: update docs for `explain analyze`.

### What is the related PR or file link(s)? 

- Reference link(s):
- This PR is to align with:
- [ ] N/A (not applicable)

**Important Notice:**

**If your changes apply to multiple TiDB documentation versions**, to trigger the bot to cherry-pick this PR to other release branches, make sure you add labels such as:

- **needs-cherry-pick-3.0**
- **needs-cherry-pick-3.1**
- **needs-cherry-pick-2.1**

### CODE REVIEW

File: reference/performance/understanding-the-query-execution-plan.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ TiDB 优化器会根据当前数据表的实际情况来选择最优的执行计
 * `loops` 是当前算子被父算子调用的次数。
 * `rows` 是当前算子返回的行的总数。例如，可以将 `count` 列的精度和 `execution_info` 列中的 `rows`/`loops` 值进行对比，据此评定查询优化器估算的精确度。
 
+除此之外，还会有 `memory` 和 `disk` 两列，会显示 root task 对应算子的内存和磁盘使用信息，如果没有使用磁盘，会显示 `N/A`。
```
Comment (XuHuaiyu) [inline reference/performance/understanding-the-query-execution-plan.md]: 保持格式不变？

\* \`memory\` 是
\*  \`disk\` 是

Comment (TomShawn) [inline reference/performance/understanding-the-query-execution-plan.md]:
```suggestion
除此之外，`EXPLAIN ANALYZE` 的输出还包括 `memory` 和 `disk` 两列，分别显示 root task 对应算子的内存和磁盘使用信息。如果没有使用磁盘，会显示 `N/A`。
```

**Comment by TomShawn** [comment]
@XuHuaiyu PTAL, again, thanks!

**Comment by qw4990** [comment]
Updated, PTAL @XuHuaiyu @TomShawn

**Pull request merged**